cookienotice/cookienotice.php

```diff
@@ -60,9 +60,8 @@ function cookienotice_addon_settings(\Friendica\App $a, &$s)
 	}
 
 	/* Add our stylesheet to the page so we can make our settings look nice */
-
-	$a->page['htmlhead'] .= '<link rel="stylesheet"  type="text/css" href="/addon/cookienotice/cookienotice.css" media="all" />' . "\r\n";
-
+    $stylesheetPath = 'addon/cookienotice/cookienotice.css';
```

Comment:
Could you please convert the spaces to TABS

Reply:
arrgh .. I dont know why, but my netbeans does it kinda randomly.

fixed